ShowerShow/DTO/UpdateDisclaimerDTO.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ShowerShow.DTO
+{
+    internal class UpdateDisclaimerDTO
+    {
+        public enum AvailableDisclaimers { get; set; }
```

Comment:
The app has a syntax error because of this so i was not even able to test the app